src/main/scala/org/broadinstitute/dsde/firecloud/model/DUOS.scala

```diff
@@ -43,31 +43,19 @@ object DUOS {
     def apply(jsValues: Map[String, JsValue]): DuosDataUse = {
       def getBoolean(f: String): Option[Boolean] = {
         jsValues.get(f) match {
-          case x if x.isDefined =>
-            Try(jsValues(f).convertTo[Boolean]) match {
-              case Success(booleanVal) => Some(booleanVal)
-              case _ => None
-            }
+          case Some(b: JsBoolean) => Try(b.convertTo[Boolean]).toOption
```

Comment:
I really don't think you need the `Try` here. If it matches as a `JsBoolean`, it will contain a Boolean inside `b.value`.